local-certs/generate-certificate.sh

```diff
@@ -11,7 +11,7 @@ chmod 600 gandi.ini
 
 # request certificate
 set -x
-certbot -n --agree-tos --email ${CERTBOT_EMAIL} ${CERTBOT_ARGS} --authenticator dns-gandi --dns-gandi-credentials gandi.ini --work-dir=$PWD/work --config-dir=$PWD/config --logs-dir=$PWD/logs -d $certificate_domains certonly
+certbot -n --agree-tos --email ${CERTBOT_EMAIL} ${CERTBOT_ARGS} --authenticator dns-gandi --dns-gandi-credentials gandi.ini --key-type rsa --work-dir=$PWD/work --config-dir=$PWD/config --logs-dir=$PWD/logs -d $certificate_domains certonly
```

Comment:
minor: do we have any control over how many bits the rsa key uses? It would be good to ensure a secure amount, though I suppose it doesn't really matter for this use case...

Reply:
We can set it with `--rsa-key-size`, but I also do not quite see the need here.
Also, we should start moving to ECDSA as soon as we get rid of the libary anyway, then the defaults are 256bit, which should be good currently.